recipes/s6-init/s6-init/init.stage1

```diff
@@ -41,7 +41,19 @@ background
   fdmove -c 2 1
 
   # Exec into stage2 script
-  /etc/rc.init $SERVICE
+  #/etc/rc.init $SERVICE
+  foreground {
+    /etc/rc.init $SERVICE
+  }
+  importas status ?
+  if -t { test $status -ne 0 }
```

Comment:
Why not simply `if -t -n { /etc/rc.init $SERVICE }` ?

Reply:
Well, partly because I don't speak execline fluently, partly because actually having the exit status printed has some value (e.g., in my case, I knew something had died from SIGABRT because this printed 134).

Reply:
Makes sense.